Source/WebCore/Modules/mediastream/RTCRtpSender.cpp

```diff
@@ -34,6 +34,7 @@
 #if ENABLE(WEB_RTC)
 
 #include "ContextDestructionObserverInlines.h"
+#include "Document.h"
```

Comment:
I wonder if you could avoid including this heavyweight header, since you don't actually interact with `Document`, it's just an object reference you pass to another function, so the compiler shouldn't need this include.